jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -109,6 +109,10 @@ then
   export SUSHY_TOOLS_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE}/resources/sushy-tools"
   export VBMC_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE}/resources/vbmc"
 
+elif [ "${REPO_NAME}" == "mariadb-image" ]
+then
+  export MARIADB_LOCAL_IMAGE="/home/${USER}/tested_repo"
+
```

Comment:
That's Jenkins configuration. Are you expecting to run integration tests for pull requests ?

Reply:
I hope so, yes. Why?

Reply:
Because title says to Prow, so I was expecting only Prow addition :)

Reply:
Ah, hmm. I guess I'm not strong in terminology :) I want it to run the absolutely normal CI.